temporal/server_option.go

```diff
@@ -124,7 +126,7 @@ func WithAudienceGetter(audienceGetter func(cfg *config.Config) authorization.JW
 // for Prometheus with framework metrics.FrameworkCustom it should be metrics.Reporter
 // not used otherwise
 // TODO: replace argument type with metrics.Reporter once tally is deprecated.
```

Comment:
Update comment. Put a one liner sample code of how to use NewOpentelemeteryReporter / NewTallyReporter to create a reporter to be supplied as input for this method.